src/Microsoft.AspNetCore.Razor.Runtime/Resources.resx

```diff
@@ -195,4 +195,16 @@
   <data name="ArgumentMustBeAnInstanceOf" xml:space="preserve">
     <value>Argument must be an instance of '{0}'.</value>
   </data>
+  <data name="TagHelperDescriptorFactory_CouldNotFindMatchingEndBrace" xml:space="preserve">
+    <value>Could not find matching ']' for required attribute.</value>
```

Comment:
We don't pass that information elsewhere when validating names. Too noisy for now. If we get push back from users on more descriptive errors we'll do it across the board